examples/no-prefix/README.md

```diff
@@ -1,3 +1,3 @@
-# Default Example
+# No Prefix Example
 
-An end to end example using the module's default variable values.
+An example similar to the basic example except in this example the prefix is hard-coded to be null. This example is used in a PR test.
```

Comment:
there is no basic example - its called default. Remove the line `This example is used in a PR test.`